tensorflow/lite/micro/kernels/kernel_util.h

```diff
@@ -27,6 +27,11 @@ limitations under the License.
 namespace tflite {
 namespace micro {
 
+TfLiteRegistration RegisterOp(
```

Comment:
Do you think this is more like RegisterOp (this function does not really register, it just create a registration that the caller will use to register)  or InitializeTfLiteRegistration or CreateTfLiteRegistration?
Realizing how big this PR is, please don't rush to make changes. I am open to discuss on this.

Reply:
The `RegisterOp` name made sense to me when I opened the PR because it's consistent with callers' convention `Register_<opname>`, i.e all `Register_<opname>` functions that exist before this PR also return a registrations but don't really register anything. I agree with you though all these are not really accurate.

I think `CreateTfLiteRegistration` makes the most sense. But if we're not going to change the callers from `Register_<opname>` to `Create<opname>TfLiteRegistration`, maybe it'll just add another source of confusion. I think there's some value in keeping a consistent name as the caller, so either `RegisterAdd() -> RegisterOp()` or `CreateAddRegistration() -> CreateRegistration()` because conceptually they're the same.

Renaming the new `RegisterOp` is not a lot of effort even though this PR is big, I'm willing to go either way.

Reply:
ok. Let us stick with with RegisterOp then. Inside Register_opname makes sense because it is directly called by tflm.